assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -35,9 +35,38 @@ const Wrapper = styled.section`
 function Panel({ initialHeight, name, children }) {
   const [isCollapsed, setIsCollapsed] = useState(false);
   const [height, setHeight] = useState(initialHeight);
+  const [manuallyChanged, setManuallyChanged] = useState(false);
 
   const collapse = useCallback(() => setIsCollapsed(true), []);
-  const expand = useCallback(() => setIsCollapsed(false), []);
+  const expand = useCallback(() => {
+    setIsCollapsed(false);
+    setHeight(initialHeight);
+  }, [initialHeight]);
+
+  useEffect(() => {
+    if (height === 0 && isCollapsed === false) {
+      collapse();
+    }
+  }, [collapse, height, isCollapsed]);
```

Comment:
This seems a bit disruptive. When you drag it too low, you can't drag it back up again, as the drag handle is now completely missing. Could we maybe only do this on release in the drag handler?

Reply:
Rethought the entire interaction, please see last commit